cni/deployments/kubernetes/install/test/install_cni.go

```diff
@@ -112,19 +112,38 @@ func rmDir(dir string, t *testing.T) {
 }
 
 // Removes Istio CNI's config from the CNI config file
-// Uses docker to execute the script due to required file permissions
-func rmCNIConfig(testNum int, cniConfigFilename string, containerID string, t *testing.T) {
+func rmCNIConfig(cniConfigFilepath string, t *testing.T) {
```

Comment:
Removed the use of `jq` in the `install-cni` e2e tests. When I added the behavior (which removes Istio CNI's config from the CNI config file) to the tests initially, I ran into some file permission errors, so I thought I had to use docker to invoke a script. But looks like that wasn't the case, and I was probably doing something wrong.